cambiatus/frontend
# Enhancement/Phone numbers

## What issue does this PR close
Closes #508, closes #510

## Changes Proposed ( a list of new changes introduced by this PR)
- Support new phone numbers:
  - [x] Portugal
  - [x] Canada
  - [x] United Kingdom
  - [x] New Zealand
- Add a better way to add new countries - just add them to `supportedCountries` in `Profile/Contact.elm`!
- Add "delete contact" button
- Let user delete contacts by just submitting empty ones in the profile edit contact page

## How to test ( a list of instructions on how to test this PR)

Test the contact modal:
1. Log in to an account that doesn't have any contact information
2. Go to the dashboard
3. See the contact modal
4. Check if the country selector (when you select either "Phone" or "Whatsapp") behaves as expected and has all the countries

Test the profile page:
1. Go to your profile
2. Go to the contact edit page
3. Check if the country selector behaves as expected and has all the countries
4. Try deleting contacts with the contact button and by leaving them blank
